src/FixedDataTableBufferedRows.js

```diff
@@ -150,6 +151,10 @@ var FixedDataTableBufferedRows = createReactClass({
 
     var baseOffsetTop = props.firstRowOffset - props.rowPositionGetter(props.firstRowIndex) + props.offsetTop;
 
+    // offset to add to rowIndex to calculate aria-rowindex
+    // aria-rowindex is 1-indexed based, but also need to add 1 for header and group header
+    var ariaIndexOffset = props.hasGroupHeader ? 3 : 2;
```

Comment:
Rather than letting `FixedDataTableBufferedRows` know about the existence of group headers, I think it's better to just pass the offset directly, which again will be calculated in `_calculateAriaAttributes`

maybe something like `props.ariaRowOffset` or `props.ariaIndexOffset`, instead of `props.hasGroupHeader`